resources/pages/login_pages.robot

```diff
@@ -5,7 +5,29 @@ Resource    ${EXECDIR}/resources/locators/login_locators.robot
 
 *** Keywords ***
 
+Adicionar o texto no campo de Pesquisa "O que você quer aprender?"
+    [Arguments]     ${text}
+    Input Text If Element Is Visible  ${INPUT_PESQUISAR}    ${text}
+
+Clicar na lupa de Pesquisa
+    Click If Element Is Visible     ${BTN_LUPA}
+
+Contar os elementos da lista
+    ${valor} =      Get Element Count   ${RESULT_LIST}
+    Log     O valor é ${valor}    console=True
+
+Clicar em um item específico do Menu da página de Login
+    [Arguments]     ${text_search}
+    ${element} =    String Replace  ${MENU_OPTION}  ${text_search}
+
+    Wait Until Element Is Visible   ${element}
+    Click If Element Is Visible  ${element}
+
+Validar se o menu está visível na tela
+    Wait Until Element Is Visible       ${MENU_BAR}
+
 Validar se a seção Entrar está visível e clicar para Entrar
+    [Arguments]     ${element_locator}
     Wait Until Element Is Visible       ${ENTRAR_LOGIN}
     Click If Element Is Visible         ${ENTRAR_LOGIN}
```

Comment:
A implementação da keyword de click já valida a visibilidade do elemento, o Wait Until Element Is Visible se faz desnecessário neste caso.

Reply:
Entendi Rafa, vou ficar mais atento!